doc/10-icinga-template-library.md

```diff
@@ -2259,6 +2259,42 @@ snmp_service_count         | **Optional.** Compare matching services with a spec
 snmp_service_showall       | **Optional.** Show all services in the output, instead of only the non-active ones. Defaults to false.
 snmp_service_noregexp      | **Optional.** Do not use regexp to match NAME in service description. Defaults to false.
 
+### Elasticsearch Netways <a id="plugin-check-command-elasticsearch-netways"></a>
+
+This category includes all subcommands using the elasticsearch-netways plugin.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                    	          | Description
+------------------------------------------|-----------------------------------------------------------------------------------------
+elasticsearch_hostname     | **Optional.** Hostname of the Elasticsearch instance (default 'localhost').
+elasticsearch_port         | **Optional.** Port of the Elasticsearch instance (default 9200).
+elasticsearch_username     | **Optional.** Username if authentication is required. No value defined as default.
+elasticsearch_password     | **Optional.** Password if authentication is required. No value defined as default.
+elasticsearch_tls          | **Optional.** Use a HTTPS connection (default false).
+elasticsearch_insecure     | **Optional.** Skip the verification of the server's TLS certificate (default false).
+elasticsearch_timeout      | **Optional.** Timeout in seconds for the CheckPlugin (default 30).
+
+#### Elasticsearch Query <a id="plugin-check-command-elasticsearch-netways-query"></a>
```

Comment:
The headers of specific check commands used to be the exact check command names.

Reply:
The rest of this file is here a good example as well.

Reply:
Rationale: Most if not all of the rest of these docs would allow me not to look at the check command config while writing service config.

In your case I -as a random user- would write `apply Service "ES Query" { check_command ="` and then copy and paste the heading `Elasticsearch Query`. I.e. `apply Service "ES Query" { check_command ="Elasticsearch Query" }`. That's not the check command name.